packages/engine/src/lib/helper/try-catch.ts

```diff
@@ -0,0 +1,28 @@
+import { ExecutionError, ExecutionErrorType } from './execution-errors'
```

Comment:
Remove this file, and in each function should use just normal shared tryCatch let's have if (isEngine(error)) {
 throw new error
}